airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/ReplicationView.tsx

```diff
@@ -91,6 +92,10 @@ const ReplicationView: React.FC<IProps> = ({ onAfterSaveSchema, connectionId })
     if (activeUpdatingSchemaMode) {
       setActiveUpdatingSchemaMode(false);
     }
+
+    if (formikHelpers) {
+      formikHelpers.resetForm({ values });
+    }
```

Comment:
Nitpick since you do this elsewhere
```suggestion
    formikHelpers?.resetForm({ values });
```